tests/test_views.py

```diff
@@ -8,36 +8,36 @@ class TestAdminViews(TestCase):
     fixtures = ['test.json']
 
     def setUp(self):
-        self.client.login(username='admin', password='password')
+        self.airtable_client.login(username='admin', password='password')
```

Comment:
In here we actually want to use `self.client` for tests. This is the test request client.